src/main/flight/pid.c

```diff
@@ -570,7 +570,7 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #ifdef USE_GPS_RESCUE
     angleTarget += gpsRescueAngle[axis] / 100.0f; // Angle is in centidegrees, stepped on roll at 10Hz but not on pitch
 #endif
-#ifdef USE_POSITION_HOLD
+#if defined(USE_POSITION_HOLD) && !defined(USE_WING)
     if (FLIGHT_MODE(POS_HOLD_MODE)) {
```

Comment:
Perhaps we can use `isFixedWing()` instead of macro here
```suggestion
#if defined(USE_POSITION_HOLD)
    if (FLIGHT_MODE(POS_HOLD_MODE) && !isFixedWing()) {
```

Reply:
If USE_WING is defined you can't use any of the autopilot even for drones now.
Once we have actual autopilot for wings, we will probably need to update isFixedWing function that it also checks the define.

Currently this line of code is correct.